doc/source/whatsnew/v0.24.0.txt

```diff
@@ -640,6 +640,7 @@ Plotting
 
 - Bug in :func:`DataFrame.plot.scatter` and :func:`DataFrame.plot.hexbin` caused x-axis label and ticklabels to disappear when colorbar was on in IPython inline backend (:issue:`10611`, :issue:`10678`, and :issue:`20455`)
 - Bug in plotting a Series with datetimes using :func:`matplotlib.axes.Axes.scatter` (:issue:`22039`)
+- The new argument ``equal_bins`` in :func:`SeriesGroupBy.hist` sets histogram bins to equal values (:issue:`22222`)
```

Comment:
This is actually `hist` method of a `SeriesGroupBy` object, although that't not available in the docs:

https://pandas.pydata.org/pandas-docs/stable/api.html

Reply:
Should it be added?

Reply:
This is available in `api.rst` if that suffices:

```
The following methods are available in both ``SeriesGroupBy`` and
``DataFrameGroupBy`` objects, but may differ slightly, usually in that
the ``DataFrameGroupBy`` version usually permits the specification of an
axis argument, and often an argument indicating whether to restrict
application to columns of a specific data type.

.. autosummary::
   :toctree: generated/

...
   DataFrameGroupBy.hist
```

Reply:
Just change the reference to DataFrameGroupBy.hist in the whatsnew - should suffice